api/tests/core/external/subcategory_suggestion_test.py

```diff
@@ -0,0 +1,40 @@
+from unittest import mock
+
+import pytest
+
+from pcapi.core.external import subcategory_suggestion
+from pcapi.core.external.subcategory_suggestion_backends import fixtures
+from pcapi.core.external.subcategory_suggestion_backends.subcategory_suggestion import SubcategorySuggestionBackend
+from pcapi.core.external.subcategory_suggestion_backends.test import TestBackend
+from pcapi.core.offerers import factories as offerers_factories
+
+
+@pytest.mark.usefixtures("db_session")
+class GetMostProbableSubcategoryIdsTest:
+    @mock.patch("pcapi.core.external.subcategory_suggestion.subcategory_suggestion_backend", TestBackend())
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏pas besoin de mock `TestBackend` Il est définit dans .env.testauto pour les tests
Je dirais même qu'il ne _faut pas_ mocker les backends